### ISSUE TITLE
Migration error in reports

### ISSUE THREAD

**Issue by jaydensmith** [barrelstrength/craft-sprout-forms#370]
### Description (screenshots, logs, ...)
Encountered a migration error in m191116_000003_add_default_entry_status_datasource_settings.php line 44.

`$formEntriesReport` has no key `reports.id`, the correct key is just `id`.

```
Exception: Undefined index: reports.id (/Users/jaydensmith/public_html/site.com/vendor/barrelstrength/sprout-forms/src/migrations/m191116_000003_add_default_entry_status_datasource_settings.php:44)
#0 /Users/jaydensmith/public_html/site.com/vendor/barrelstrength/sprout-forms/src/migrations/m191116_000003_add_default_entry_status_datasource_settings.php(44): modules\zoho\services\Service->modules\zoho\services\{closure}(8, 'Undefined index...', '/Users/jaydensm...', 44, Array)
#1 /Users/jaydensmith/public_html/site.com/vendor/craftcms/cms/src/db/Migration.php(58): barrelstrength\sproutforms\migrations\m191116_000003_add_default_entry_status_datasource_settings->safeUp()
#2 /Users/jaydensmith/public_html/site.com/vendor/craftcms/cms/src/db/MigrationManager.php(239): craft\db\Migration->up(true)
#3 /Users/jaydensmith/public_html/site.com/vendor/craftcms/cms/src/db/MigrationManager.php(159): craft\db\MigrationManager->migrateUp(Object(barrelstrength\sproutforms\migrations\m191116_000003_add_default_entry_status_datasource_settings))
#4 /Users/jaydensmith/public_html/site.com/vendor/craftcms/cms/src/services/Updates.php(225): craft\db\MigrationManager->up()
#5 /Users/jaydensmith/public_html/site.com/vendor/craftcms/cms/src/console/controllers/MigrateController.php(265): craft\services\Updates->runMigrations(Array)
#6 [internal function]: craft\console\controllers\MigrateController->actionAll()
#7 /Users/jaydensmith/public_html/site.com/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#8 /Users/jaydensmith/public_html/site.com/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#9 /Users/jaydensmith/public_html/site.com/vendor/yiisoft/yii2/console/Controller.php(148): yii\base\Controller->runAction('all', Array)
#10 /Users/jaydensmith/public_html/site.com/vendor/yiisoft/yii2/base/Module.php(528): yii\console\Controller->runAction('all', Array)
#11 /Users/jaydensmith/public_html/site.com/vendor/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction('migrate/all', Array)
#12 /Users/jaydensmith/public_html/site.com/vendor/craftcms/cms/src/console/Application.php(93): yii\console\Application->runAction('migrate/all', Array)
#13 /Users/jaydensmith/public_html/site.com/vendor/yiisoft/yii2/console/Application.php(147): craft\console\Application->runAction('migrate/all', Array)
#14 /Users/jaydensmith/public_html/site.com/vendor/yiisoft/yii2/base/Application.php(386): yii\console\Application->handleRequest(Object(craft\console\Request))
#15 /Users/jaydensmith/public_html/site.com/craft(22): yii\base\Application->run()
#16 {main}
```

**Comment by BenParizek** [comment]
Thanks @jaydensmith.

We'll get this corrected.

**Comment by BenParizek** [comment]
@jaydensmith This has been corrected in Sprout Forms v3.6.2
